src/Sylius/Bundle/UserBundle/EventListener/UserLastLoginSubscriber.php

```diff
@@ -72,7 +72,7 @@ public function onImplicitLogin(UserEvent $event)
     /**
      * @param UserInterface $user
```

Comment:
@lchrusciel It could be changed to object. But maybe it's more useful to let UserInterface for IDE ?